eventkit_cloud/tasks/task_runners.py

```diff
@@ -241,57 +85,49 @@ def run_task(self, provider_task_uid=None, user=None, run=None, stage_dir=None,
+                export_provider_task=export_provider_task_record, worker=worker,
+                display=getattr(task.get('obj'), "display", False)
+            )
+            export_tasks[format]['task_uid'] = export_task.uid
 
         """
-        Create a celery chain which uses a gpkg file with the planet osm schema, so that it can be converted to a
-        thematic schema.
+        Create a celery chain which gets a gpkg of osm data & runs export formats
         """
-        thematic_gpkg_task = create_format_task('gpkg-thematic')
-        thematic_gpkg = {'obj': thematic_gpkg_task,
-                         'task_uid': create_export_task(task_name=thematic_gpkg_task.name,
-                                                        export_provider_task=export_provider_task, worker=worker, display=getattr(thematic_gpkg_task, "display", False)).uid}
-
-        # Note this needs to be mutable (s instead of si) so that it can take the result of the generic osm tasks.
-        thematic_task = thematic_gpkg.get('obj').s(
-            run_uid=run.uid, stage_dir=stage_dir, job_name=job_name, task_uid=thematic_gpkg.get('task_uid'),
-            user_details=user_details
-        ).set(queue=worker, routing_key=worker)
-
         if export_tasks:
             format_tasks = chain(
                 task.get('obj').s(
                     run_uid=run.uid, stage_dir=stage_dir, job_name=job_name, task_uid=task.get('task_uid'),
                     user_details=user_details
                 ).set(queue=worker, routing_key=worker)
-                for task_name, task in export_tasks.iteritems()
+                for format_ignored, task in export_tasks.iteritems()
             )
         else:
             format_tasks = None
 
-        thematic_tasks = (thematic_task | format_tasks) if format_tasks else thematic_task
-
         bbox = run.job.extents
-        style_task = osm_create_styles_task.s(task_uid=create_export_task(task_name=osm_create_styles_task.name,
-                                                                           export_provider_task=export_provider_task,
-                                                                           worker=worker, display=getattr(osm_create_styles_task, "display", False)).uid,
-                                              stage_dir=stage_dir,
-                                              job_name=job_name,
-                                              bbox=bbox,
-                                              provider_slug=provider_task.provider.slug,
-                                              provider_name=provider_task.provider.name,
-                                              user_details=user_details).set(queue=worker, routing_key=worker)
 
-        return export_provider_task.uid, (thematic_tasks | style_task)
+        osm_gpkg_task = osm_data_collection_task.si(
+            stage_dir, export_provider_task_record.id, worker=worker, osm_query_filters=job.filters,
```

Comment:
Since we aren't filtering in overpass.py these filters no longer need to be passed, as the variable seems to be unused in the subsequent calls.